### PR TITLE
Settings docs update

### PR DESCRIPTION
**Pull request by ryestew**

### CODE REVIEW

File: docs/settings.md [hunk 1]
```diff
@@ -3,12 +3,27 @@ Settings
 
 To get to **Settings** click the gear a the very bottom of the icon panel.
```
Comment (Aniket-Engg) [inline docs/settings.md:4]: typo: the gear `at` the very bottom

File: docs/settings.md [hunk 2]
```diff
@@ -3,12 +3,27 @@ Settings
...
-![](images/a-themes.png)
+![](images/a-settings-general.png)
 
-Another important settings:
+-  Generate contract metadata is used for deploying with libraries.  See our blog post on the subject: [Deploying with Libraries](https://medium.com/remix-ide/deploying-with-libraries-on-remix-ide-24f5f7423b60?sk=68f9c2bf318e85e129e46fe44683a730)
 
--   Text wrap: controls if the text in the editor should be wrapped.
+-  Text wrap: controls if the text in the editor should be wrapped.
+-  Personal mode: can be used when one is connecting to a **local node**.  It is used to have Remix temporarily save the passphrase - so that you don't need to **unlock** the account in GETH.  Remix will not persist the passphrase - so if you refresh your browser the passphrase will be gone.
+-  Matomo Analytics: This is where you can turn off and on your approval for us to use Matomo.  We do not collect any personally identifyable information (PII) and our reports are public.  See our [blog post on the subject](https://medium.com/remix-ide/help-us-improve-remix-ide-66ef69e14931?source=friends_link&sk=cf9c62fbe1270543eb4bd912e567e2d6).
```
Comment (Aniket-Engg) [inline docs/settings.md:14]: typo: `identifyable` should be `identifiable `

File: docs/settings.md [hunk 3]
```diff
@@ -3,12 +3,27 @@ Settings
...
 
-- Enable optimization: defines if the compiler should enable optimization during compilation. Enabling this option saves execution gas. It is useful to enable optimization for contracts ready to be deployed in production but could lead to some inconsistencies when debugging such a contract.
+
+
+
+
+### Github Access Token
+
+For creating GISTs as well as other Github functions, it is necessary to put in an access token.  [https://github.com/settings/tokens](https://github.com/settings/tokens)
```
Comment (Aniket-Engg) [inline docs/settings.md:22]: as well as `using` other Github

**Review by Aniket-Engg — COMMENTED** [review]
`Always use Ethereum VM at Load` from settings tab is not explained

**Review by Hbutlercapone — APPROVED** [review]

**Comment by Aniket-Engg** [comment]
> `Always use Ethereum VM at Load` from settings tab is not explained

@ryestew Can you please consider this too

**Review by LianaHus — APPROVED** [review]

**Pull request merged**